### PR TITLE
Do not run if node version older than 5.0

### LINKED ISSUE #105: CLI should warn if Node version < 5.x
(orig reported at #73 )

Since we're using a bunch of es2015 features, node 4 isn't supported.

The cli should error and exit if you're not running using a 5.x runtime.

question: Should the error message recommend using nvm?

### PR DESCRIPTION
**Pull request by deepanjanroy**
Fixes #105.

### CODE REVIEW

**Comment by paulirish** [comment]
this wfm!

File: cli.js [hunk 1]
```diff
@@ -38,6 +39,11 @@ const cli = meow(`
 const defaultUrl = 'https://m.flipkart.com';
 const url = cli.input[0] || defaultUrl;
 
+if (semver.lt(process.version, '5.0.0')) {
```
Comment (samccone) [inline cli.js]: let's use 5.9 here :)

**Comment by samccone** [comment]
Once the comment is taken care of we can squash this down to 1 commit  :)

**Comment by deepanjanroy** [comment]
Updated to 5.9 and sqashed to one commit :sparkles:

**Comment by samccone** [comment]
boom thanks @deepanjanroy :clap:

**Pull request merged**